src/home/NotificationCenter.test.tsx

```diff
@@ -940,4 +949,109 @@ describe('NotificationCenter', () => {
       })
     })
   })
+
+  describe('clevertap notifications', () => {
+    it('renders clevertap notification', () => {
+      const store = createMockStore({
+        ...storeDataNotificationsDisabled,
+        home: {
+          cleverTapInboxMessages: [mockExpectedCleverTapInboxMessage],
+        },
+      })
+      const { getByText } = render(
+        <Provider store={store}>
+          <NotificationCenter {...getMockStackScreenProps(Screens.NotificationCenter)} />
+        </Provider>
+      )
+      expect(getByText('CleverTap Message Header')).toBeDefined()
+      expect(getByText('CleverTap Message Body Text')).toBeDefined()
+    })
+
+    it('emits correct events when CTA is pressed', () => {
+      const store = createMockStore({
+        ...storeDataNotificationsDisabled,
+        home: {
+          cleverTapInboxMessages: [mockExpectedCleverTapInboxMessage],
+        },
+      })
+      const { getByText } = render(
+        <Provider store={store}>
+          <NotificationCenter {...getMockStackScreenProps(Screens.NotificationCenter)} />
+        </Provider>
+      )
+
+      fireEvent.press(getByText('CleverTap Message CTA'))
+      expect(store.getActions()).toEqual([
+        fetchAvailableRewards(),
```

Comment:
nit: this feels unrelated, you could assert instead that getActions() was last called with openUrl or you could do store.clearActions() after the render instead, to keep the tests specific to notifications?